src/api.rs

```diff
@@ -5,8 +5,8 @@ use crate::{
     SyntaxKind, SyntaxText, TextRange, TextSize, TokenAtOffset, WalkEvent,
 };
 
-pub trait Language: Sized + Clone + Copy + fmt::Debug + Eq + Ord + std::hash::Hash {
-    type Kind: fmt::Debug;
+pub trait Language: Sized + Copy + fmt::Debug + Eq + Ord + std::hash::Hash {
+    type Kind: Sized + Copy + fmt::Debug + Eq + Ord + std::hash::Hash;
```

Comment:
Why do we need these extra bounds now?

Reply:
I see, the bounds make sense for this assoc type anyways I'd say so that's fine.